content/4_Gold/Unweighted_Shortest_Paths.mdx

```diff
@@ -248,6 +248,65 @@ public class Solution {
 
 </JavaSection>
 
+<PySection>
+
+```py
+from collections import deque, defaultdict
+
+
+def find_shortest_path(n, m, edges):
```

Comment:
does this need to be in a function?

Reply:
no, it still works, so i updated the code.

![image](https://github.com/user-attachments/assets/02cac864-295c-4da9-bbca-b5cadc0a9a27)